CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 0.13.0
```

Comment:
This should really go into the "Unreleased" section for now.

Reply:
Fixed!